lib/node_modules/@stdlib/math/base/assert/int32-is-even/benchmark/benchmark.js

```diff
@@ -31,14 +30,19 @@ var isEven = require( './../lib' );
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
+	var opts;
 	var x;
 	var y;
 	var i;
 
+	opts = {
+		'dtype': 'float32'
```

Comment:
@hrshya This isn't correct. The previous code cast to `int32`, not `float32`. You should use `discrete-uniform` and set the output dtype to int32. Applies here and in the `native.js` benchmark file.

Reply:
The previous code was a bit confused in that the previous import was for `to-float32`, but this should have been `to-int32`.